simulation/src/test/java/com/palantir/dialogue/core/Strategy.java

```diff
@@ -54,114 +48,69 @@ public Channel getChannel(Simulation simulation, Supplier<List<SimulationServer>
     }
 
     private static Channel concurrencyLimiter(Simulation sim, Supplier<List<SimulationServer>> channelSupplier) {
+        Random pseudo = new Random(3218974678L);
         return RefreshingChannelFactory.RefreshingChannel.create(channelSupplier, channels -> {
-            List<LimitedChannel> limitedChannels = channels.stream()
-                    .map(addConcurrencyLimiter(sim))
-                    .map(addFixedLimiter(sim))
-                    .collect(Collectors.toList());
-            LimitedChannel limited1 = new RoundRobinChannel(limitedChannels);
-            return retryingChannel(sim, limited1);
-        });
-    }
-
-    private static Channel concurrencyLimiterBlacklistRoundRobin(
-            Simulation sim, Supplier<List<SimulationServer>> channelSupplier) {
-        return RefreshingChannelFactory.RefreshingChannel.create(channelSupplier, channels -> {
-            List<LimitedChannel> limitedChannels = channels.stream()
-                    .map(addConcurrencyLimiter(sim))
-                    .map(addFixedLimiter(sim))
-                    .map(c -> new BlacklistingChannel(c, Duration.ofSeconds(1), sim.clock()))
-                    .collect(Collectors.toList());
-            LimitedChannel limited1 = new RoundRobinChannel(limitedChannels);
-            return retryingChannel(sim, limited1);
+            return DialogueChannel.builder()
+                    .channels(channels)
+                    .clientConfiguration(ClientConfiguration.builder()
+                            .from(stubConfig())
+                            .taggedMetricRegistry(sim.taggedMetrics())
+                            .nodeSelectionStrategy(NodeSelectionStrategy.ROUND_ROBIN)
+                            .failedUrlCooldown(Duration.ofMillis(200))
```

Comment:
Do we consume `failedUrlCooldown` anywhere?

Reply:
Don't think so, but you can't construct a ClientConfiguration instance without specifying it!!

Reply:
Ah I see, thanks! Looks like the default in `ClientConfigurations.java` is `Duration.ZERO`, we may want to emulate that.

Reply:
In a follow-up we could use positive values to wrap delegates with a BlacklistingChannel inside of RoundRobinChannel.

Reply:
Mmm looks like we can't get away with zero:
```
SafeIllegalArgumentException: If nodeSelectionStrategy is ROUND_ROBIN then failedUrlCooldown must be positive
	at com.palantir.logsafe.Preconditions.checkArgument(Preconditions.java:53)
	at com.palantir.conjure.java.client.config.ClientConfiguration.check(ClientConfiguration.java:132)
```